CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # browser-switch-android Release Notes
 
+## unreleased
+* Update dependencies
```

Comment:
Should/can this be more specific?

Reply:
Normally it doesn't seem like we add changelog entries for updating dependencies, but since we are going to release this and there aren't any other unreleased changes currently, we wanted to add something for the release notes. I've updated the entry to be more specific.

Reply:
Also seems like updating dependencies has been more of an implicit process in previous versions.